app/Libraries/BBCodeFromDB.php

```diff
@@ -85,11 +85,19 @@ public function parseBoxHelperSuffix()
 
     public function parseCentre($text)
     {
-        $text = str_replace("[centre:{$this->uid}]", '<center>', $text);
-        $text = str_replace("[/centre:{$this->uid}]", '</center>', $text);
+        $text = str_replace("[centre:{$this->uid}]", '<div style='text-align:center;'>', $text);
```

Comment:
I didn't check the other one but this one doesn't center quite a few elements (imagemap, spoilerbox, audio, video, and maybe others)

also this code as is isn't valid php (incorrect quoting).